Code of conduct 2024 October revisions

anne-decusatis opened the pull request:
The following dates in my September draft are still relevant: 

2024-09-13 (Friday): “October policy text” has a denhac-wide location open to comments, and all content moderation issues have found a viable resolution 
2024-10-01 (Tuesday): October board meeting, public discussion and feedback can be given
2024-10-15 (Tuesday), or one month after the comments open if target is not met: October doc closes to new comments but existing ones may still be in progress
2024-10-29 (Tuesday): October document is final, all comments have been resolved
2024-11-05 (Tuesday): November board meeting discussion, and decision to vote or withdraw on that section 

Additions and changes can still continue in the future as there is desire or need

anne-decusatis committed:
Add "guidelines" and feedback from live meeting

This closes out the September document https://docs.google.com/document/d/1buC_WPA5qJc3iBO3syr-d0QdDQh3aUZ4F793peg2ccs/edit

anne-decusatis committed:
Additional notes on process changes that are included in this proposal

Jnesselr commented on code-of-conduct-2024-addendum.md:
```diff
@@ -0,0 +1,27 @@
+  * When?
+  * Where?
+  * How?
+* If there is any action you'd like to request that the board takes in response to this specific incident - let us know here.
```
I would be clear that just because something is requested doesn't mean that the board will do that thing. If this is level setting from the side of the person reporting the incident to the board, then there should be some level setting in the other direction to prevent hurt feelings.

anne-decusatis replied:
I agree with that point and that there should be level-setting. I imagined that level-setting being more of an in-person discussion - it feels kind of discouraging to me to say "Hey, what do you want to happen, btw we aren't promising not to ignore what you want if we have other obligations that require us to take specific actions" during the initial report - but maybe it's better to have it be known in advance since it's ultimately true anyway? There's probably some way to say it that doesn't discourage reporting?

anne-decusatis replied:
I thought about this more, I think it is better to be transparent in advance that while the intent is to respect a reporter's autonomy, not all requested actions are possible. Edited, resolving this but please let me know if it isn't resolved.

Jnesselr commented on code-of-conduct.md:
```diff
@@ -12,21 +12,33 @@ We expect everyone involved in our community to follow this code of conduct. Thi
+Some guidelines on appropriate behavior:
+
+* If you are unsure if something is appropriate behavior, err on the side of caution. 
+* Each person you interact with can define what is appropriate for them. 
+* Impact matters more than intent. 
+* It is your responsibility to ensure that your behavior does not have a negative impact.
+* Problems happen when we assume that our way of thinking or behaving is the norm or ok with everyone. This is particularly problematic when we are in a position of power or privilege.
```
I think these lines are what I have the most issue with out of these proposed changes. They feel like they're written with good intent but haven't been battle tested with really crappy people who would abuse them.

Things like "impact matters more than intent" sound great, but the reality is they both matter and there's lots of situations where you do need to understand intent as a lens for the impact. If someone gets hit in the head with a board, the first questions will be all about context, who was paying attention, whether someone was being negligent, etc. Similarly, "each person you interact with can define what is appropriate for them" can be abused pretty easily. "Hey I think rainbows are disrespectful to Christians, so I don't like that you're displaying rainbows" for example.

I'm wondering if it wouldn't be better to focus on things that people might not see as problematic, but end up being problematic. For example, if you're a woman in the space and every other guy comes up and offers you advice, but they wouldn't do that for other men in the space, that's not harassment per se, but it is objectively annoying and it's a common thing people don't think about.

anne-decusatis replied:
I think this is a great point, thank you for sharing it. 

I am wondering if the concern about bad actors can be addressed by rewording these points, or if what we need is a different framework/complete rewrite of this section. I'm fine with either way to move forward and might experiment with both, but value your input if you have specific thoughts on it. I am also wondering if this should be added to the code of conduct directly or if it should be a separate document with potentially more examples and an anti-harassment-training tone.

kravlin replied:
This is the only section I think needs rewrites. 

There's a reason a lot of corporate CoCs leave out guidelines like this, and that's because a lot of the guidelines end up being interpreted as gospel and used as rules.

Saying something like "denhac's membership is diverse and comes from different backgrounds, histories and experiences. Problems may occur when we assume that everyone shares our viewpoints or ways of thinking. Careless words and actions can affect other people in unintended ways. We expect all members to treat other members respectfully to promote a positive atmosphere within the space."

This doesn't tell people exactly how to treat other people but instead warns people of the dangers of not doing the things in the bullet points above while not explicitly giving power to distinct actions?

I think we can re-write this to not provide a bulleted list of

Jnesselr commented on code-of-conduct.md:
```diff
@@ -12,21 +12,33 @@ We expect everyone involved in our community to follow this code of conduct. Thi
-11. Deliberate “outing” of a sensitive aspect of a person’s identity without their consent
-12. Deliberate misgendering. This includes dead-naming or persistently using a pronoun that does not correctly reflect a person’s gender identity
-13. Retaliating against anyone who files a complaint that someone has violated this code of conduct 
+3. Physical contact, romantic advances, or sexual attention that continues after a request to stop or a negative/"no" response.
```
Or lack of affirmative "yes". These guidelines will always be written for people who are bad at social cues and need the rules to have as a reference guide.

anne-decusatis replied:
In this section I made minor readability edits to pts 3 and what-is-now-12-but-was-11. I also, to address the issue you're commenting on around affirmative consent (which was also brought up in those words in Tuesday's meeting), added point 4 (which moved points 4-13 into 5-14 in the diff, making it difficult to read on Github default settings)

My thought is, which I believe was mentioned in the public board meeting by someone else as well - the intent is not to prohibit a member being asked out by another member once in a respectful manner, and it's not easy to obtain "yes you can ask me on a date" until you have actually done the asking-out (as one example). Calling it "unwanted" is likewise not necessarily possible to judge in advance. So I wanted to break this up into two points, where all repeated behavior after being told to stop is prohibited (though we do call out touch and romantic/sexual behaviors as especially risky), and for the behaviors in that list where it is always possible to receive consent in advance, you have to receive consent. 

I also worded new-4 as "informed and voluntary" consent because I couldn't find a straightforward, short, and applicable definition of "affirmative consent" already written on a web search (I didn't look that thoroughly though), but I saw those words come up repeatedly in the definitions I did find. I do not think everyone has the same ideas in their heads of what affirmative consent means; I think informed and voluntary are more specific. Other adjective suggestions are welcome!

One additional thought on this is that obviously sexual activity is already banned at denhac in several other places in this document, so this would be aimed mostly at members interacting outside the space. I would like to have the latitude to address reports of a member violating our code of conduct outside the space via formal reporting procedures in egregious cases which impact space safety (though I don't think it is necessarily something we want to touch if it doesn't impact the safety of our space/of a significant part of our membership) - I am not sure if/how to clarify that, or even if it is something others agree with me on.

anne-decusatis reviewed (commented):
response to comments - thank you @Jnesselr for the thoughtful feedback! I intend on adding a new commit with fixes based on your feedback after I've figured out the changes to make in response, wanted to give a chance for a little more back and forth first.

anne-decusatis committed:
Clarify when "we" is the board and when "we" is all members

anne-decusatis committed:
Change form language around requested actions

Commit:
break out guidelines into a separate best practices

Commit:
tweaks to code of conduct based on feedback

mvan13 commented on code-of-conduct-2024-addendum.md:
```diff
@@ -0,0 +1,27 @@
+
+The following should be read to every new member when they join, in addition to the existing practices and guidelines that appear when a key card is activated:
+
+* No Harassment: This includes, but is not limited to, attention that comes after a request to stop. Our anti-harassment policy is part of our member agreement - please read it.
```
example comment

Commit:
minor wording tweaks

Commit:
reword based on feedback to be more concise & direct

also broke out examples into "questions to ponder" and "examples" which give more direct examples
thanks Leah for the feedback and suggested rewords!
`

anne-decusatis commented on code-of-conduct.md:
```diff
@@ -2,47 +2,48 @@
-* Membership will be suspended for a length of time.
-* Membership will be terminated.
+* The member will be put on probation and removed from good standing.
+* Membership will be suspended for a predefined length of time, often until the next public board meeting. This can be done by any one board member or manager if there is a safety concern.
```
live feedback: replace with link to bylaws sections which already define this, rather than duplicating

anne-decusatis commented on code-of-conduct-best-practices.md:
```diff
@@ -0,0 +1,51 @@
+
+## Purpose
+
+This document outlines best practices for behavior at denhac, complementing our existing code of conduct. It emphasizes each member's responsibility to assess their intent, action, and impact. The code of conduct establishes minimum standards for acceptable conduct, identifies unacceptable behavior, and describes the procedures for reporting such behavior to the board of directors. denhac's goal  ...
```
live feedback: the Best Practices should say something about how it the CoC supersedes it

anne-decusatis commented on code-of-conduct.md:
```diff
@@ -2,47 +2,48 @@
 
```
live feedback: should have "you" replaced with "member" or define it or somewhere since this is legal

anne-decusatis commented on code-of-conduct-best-practices.md:
```diff
@@ -0,0 +1,51 @@
+# Best practices for interaction at denhac
```
if we vote, do we need to vote to change it too? specify

anne-decusatis reviewed (commented):
live feedback

anne-decusatis commented on code-of-conduct.md:
```diff
@@ -2,47 +2,48 @@
+8. Deliberate intimidation by words, gestures, body language, or menacing behavior
+9. Stalking
+10. Harassing photography or recording, including logging online activity for harassment purposes
+11. Continued one-on-one contact or communication after active requests to cease, multiple instances of no response, or other indicators of a lack of consent
```
would love feedback on how to make this more actionable / indicative of a pattern of behavior

Commit:
Address comments from live

reduce "you" usage; link to bylaws instead of repeating; address conflicts with CoC and best practices; does not require vote on all edits

Commit:
feedback: add example of trainers to power

Commit:
reword pt 11 around non-response as a lack of consent to make it more about a pattern of behavior

anne-decusatis commented on code-of-conduct.md:
```diff
@@ -2,48 +2,50 @@
+8. Deliberate intimidation by words, gestures, body language, or menacing behavior
+9. Stalking
+10. Harassing photography or recording, including logging online activity for harassment purposes
+11. Continued one-on-one contact or communication after active requests to cease or other indicators of a lack of consent (such as a pattern of sending multiple long messages that are not necessary to space operations and that receive no response)
```
still want feedback on this line

chungl replied:
I don't have a strong opinion on the inclusion of this new example. On one hand, it's the only example scenario included in this section, and we have a new "best practices" document where this may arguably fit better. On the other hand, it does clarify intent and scope of this rule in a way that would otherwise not be obvious.

I don't object to including it. I do think we would be able to address such a issue without this parenthetical text, but perhaps this preempts some undesirable behavior and/or some of the debate and blowback stemming from an enforcement action.

eddiezane replied:
I think that the latter half of this point belongs in the best practices doc because it is too ambigous.

Including this does not put enough responsibility on people to solve their problems by either communicating directly or by using the escalation pathway of going through the board. I can see a future where this creates a signifiant number of reports that could have been handled with a simple converstation and instead needs an incident response process.

chungl commented on code-of-conduct-2024-addendum.md:
```diff
@@ -0,0 +1,27 @@
+The following should be read to every new member when they join, in addition to the existing practices and guidelines that appear when a key card is activated:
+
+* No Harassment: This includes, but is not limited to, attention that comes after a request to stop. Our anti-harassment policy is part of our member agreement - please read it.
+* No Bullying: Recognize and respect everyone’s differences. Treat all members with kindness and respect.
```
This isn't a super strong opinion, but I'd prefer to collapse these into one point. There's a limit to what people can retain when they're read a wall of text, and there's a lot of other important information we convey here, too.

chungl commented on code-of-conduct.md:
```diff
@@ -2,48 +2,50 @@
 denhac does not promote or discriminate against any person, population group, or organization with regard to categories protected by applicable United States law. These include, but are not limited to race, color, religion, sex, gender expression, sexuality, physical appearance, language, education background, national origin, age, disability, and veteran status.
 
-denhac strives to create an inclusive environment where everyone can feel welcome and respected in our space. We know that, while there is not a formal hierarchy here as in a workplace, everyone comes into our space with their own different histories and social norms. We expect you to both communicate with each other about the boundaries/limits you wish to hold, and respect when someone else comm ...
+denhac strives to create an inclusive environment where everyone can feel welcome and respected in our space. While there is not a formal hierarchy here as in a workplace, everyone comes into our space with their own different histories and social norms. We on the Board of Directors expect members to both communicate with each other about the boundaries/limits you each wish to hold, and respect w ...
```
> oversteps a boundary that is clearly expressed to them

I feel like this should be `that has been clearly expressed to them`

> the board may need to take disciplinary action

I think it would be more appropriate to strike `need to` and simply state `the board may take disciplinary action`

chungl replied:
I don't see any requirement that members document communicated boundaries, or any suggestion thereof in the best practices. Ideally this isn't a requirement, but is it going to practically limit enforceability? What happens when someone says they verbally communicated a boundary, and the defendant denies or disputes the communication?

This is likely something that needs to be addressed in the next revision cycle at this point.

chungl commented on code-of-conduct.md:
```diff
@@ -2,48 +2,50 @@
+denhac strives to create an inclusive environment where everyone can feel welcome and respected in our space. While there is not a formal hierarchy here as in a workplace, everyone comes into our space with their own different histories and social norms. We on the Board of Directors expect members to both communicate with each other about the boundaries/limits you each wish to hold, and respect w ...
 
-If you feel that you have been harassed at denhac or if someone has made you uncomfortable (even if you aren’t sure if it’s harassment), we encourage you to follow the reporting procedure in our code of conduct. We want to make the space better and creating a safe environment for you is important to making denhac better - but we can’t act to address potential issues if we aren’t made aware of the ...
+If you feel that you have been harassed at denhac or if someone has made you uncomfortable (even if you aren’t sure if it’s harassment), we on the Board of Directors encourage you to follow the reporting procedure in our code of conduct. Everyone can help make denhac better, and creating a safe environment for everyone is important to making denhac better. Board members can’t act to officially ad ...
```
> Board members can’t act to officially address potential issues if we aren’t made aware of them.

I'd like to strike `officially`; we can't address issues we aren't aware of.

chungl reviewed (commented):
This is a great step forward in clearly articulating our expectations. Thank you Anne for taking on the significant work of coordinating this project, and thank you to everyone who provided feedback and participated in this project.

I've made a few small wording recommendations, none of which are blocking or worthy of further amendment on their own.

My only serious concern is whether we need to request/suggest documentation of boundary setting, in order to make violations more expeditiously actionable. That question is probably out of scope for this release cycle.

_I'm not "approving" this request since it has not yet been voted on._

eddiezane commented on code-of-conduct-2024-addendum.md:
```diff
@@ -0,0 +1,27 @@
+
+## Changes to new joiner messaging
+
+The following should be read to every new member when they join, in addition to the existing practices and guidelines that appear when a key card is activated:
```
https://github.com/Denhac/denhac-webhooks/blob/d7f223e6b6d42dc35ed29f01ab9d2e72ccaaf741/app/External/Slack/Modals/NewMemberInfoModal.php#L15

eddiezane commented on code-of-conduct-best-practices.md:
```diff
@@ -0,0 +1,53 @@
+
+### Seeking and receiving informed consent in advance
+
+For behaviors where consent can be obtained in advance, it is essential to do so beforehand. This applies to all interactions, not just those deemed risky, such as romantic or sexual advances.
```
I find this to be a little too vague and I'm not trying to be pedantic. I understand the intent, but the wording as it is now suggests that you should seek consent for just about anything - like saying hello.

anne-decusatis reviewed (commented):
approved 4-2 as-is on Nov 05: https://denhac.org/board-minutes/board-minutes-2024-11-05
